ampeg/src/test/java/ManualTestCases.java

```diff
@@ -0,0 +1,73 @@
+import static org.testng.Assert.assertEquals;
+
+import java.io.BufferedInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.text.PDFTextStripper;
+
+public class ManualTestCases extends AmpegTests{
```

Comment:
These are no longer manual test cases. Should rename to something more descriptive.